samples/csharp/getting-started/BinaryClassification_SentimentAnalysis/SentimentAnalysis/SentimentAnalysisConsoleApp/Program.cs

```diff
@@ -103,6 +105,25 @@ private static void TestSinglePrediction(MLContext mlContext)
             Console.WriteLine($"==================================================");
         }
 
+        private static void DownloadDataIfNotExists(string dataPath)
+        {
+            var downloadPath = "http://aka.ms/tlc-resources/benchmarks/WikiDetoxAnnotated160kRows.tsv";
```

Comment:
Likely my fault at somepoint in the past for giving a HTTP link. 
```suggestion
            var downloadPath = "https://aka.ms/tlc-resources/benchmarks/WikiDetoxAnnotated160kRows.tsv";
```